src/support/z_utils.erl

```diff
@@ -255,17 +244,6 @@ hex_encode(Value) -> z_url:hex_encode(Value).
 hex_decode(Value) -> z_url:hex_decode(Value).
 
 
-%%% URL ENCODE %%%
-url_encode(S) -> z_url:url_encode(S).
-url_decode(S) -> z_url:url_decode(S).
-url_path_encode(L) -> z_url:url_path_encode(L).
-percent_encode(S) -> z_url:percent_encode(S).
-
-url_valid_char(C) -> z_url:url_valid_char(C).
-url_reserved_char(C) -> z_url:url_reserved_char(C).
-url_unreserved_char(C) -> z_url:url_unreserved_char(C).
-
-
 %% @spec os_filename(String) -> String
 %% @doc Simple escape function for filenames as commandline arguments.
 %% foo/"bar.jpg -> "foo/\"bar.jpg"; on windows "foo\\\"bar.jpg" (both including quotes!)
```

Comment:
Line 249 is too long: %% foo/"bar.jpg -> "foo/\"bar.jpg"; on windows "foo\\\"bar.jpg" (both including quotes!).